index.html

```diff
@@ -6,11 +6,67 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Responsive Cake webpage</title>
-    <!-- Add a link to your css file here -->
+    <link rel="stylesheet" href="/style.css">
+
 </head>
 
 <body>
-    <!-- Add your markup here -->
+    
+    <main>
+        <nav>
+<!-- web page logo and nav bar as burger menu-->
+            <div class="logo">IMG</div>
+            <ul class="navbar">
+                <li>Home</li>
+                <li>Cakes</li>
+                <li>Ordering</li>
+                <li>Lessons</li>
+                <li>About</li>
+            </ul>
+        </nav>
+
+<!---slide show of cake images with hello overlaying --->
+        <section class="carouselcakes">
+            <div class="slidecontainercake">
+                <div class="pic1">PIC1</div>
+                <div class="pic2">PIC2</div>
+                <div class="pic3">PIC3</div>
+                <div class="pic4">PIC4</div>
+            </div>
+            
+            <section class="hello">
+            <h1>HELLO</h1>
+        </section>
+        </section>
```

Comment:
Remember to keep an eye on your formatting, the indentation on the page should show how nested an element is in orded to clearly see what html element is a child of another.